### ISSUE TITLE
Add “Resources” section in the Airflow website

### ISSUE THREAD

**Issue by aijamalnk** [apache/airflow-site#233]
Add “Resources” section in the community page after “Join the community” and before “Primary Members Commiters”. Also add Menu to the “Community” page that shows what is on the page to the users at a glance. Here is a good example https://beam.apache.org/community/presentation-materials/

- This place will have logos, and other promo materials https://cwiki.apache.org/confluence/display/AIRFLOW/File+lists
- This place will have the Airflow book https://www.manning.com/books/data-pipelines-with-apache-airflow

**Comment by Qay-Tekkie** [comment]
@aijamalnk Please can I work on this issue? I'm an Outreachy applicant.

**Comment by XD-DENG** [comment]
Hi @Qay-Tekkie Thanks for your interest. You're more than welcome to contribute. You can directly proceed to prepare a PR I think (and comment here then maybe).

**Comment by Lokeshrathi** [comment]
Is someone working on this?

**Comment by potiuk** [comment]
Not that I am awarey of - but feel free. Shall I assign you?

**Comment by Lokeshrathi** [comment]
Can you give me more specifics on this, I am new here and would love to learn. @potiuk

**Comment by potiuk** [comment]
Not really. This issue has been opend in 2020 as you see and this was an idea. And part of the issue is to make a proposal what to do. So it's up to a person to figure out best ways of addressing the issue proposed. If unsure - don't do the task.

**Comment by divyaguduru** [comment]
hi @aijamalnk , i have gone through the description and would like to pick this issue .. pls assign it to me

**Comment by gdivya6028** [comment]
Resource section content is not displayed on [Apache Airflow website](https://airflow.apache.org/community/#community-resources), however in local when testing with main branch can see the Resource section content 

<img width="1321" alt="image" src="https://github.com/user-attachments/assets/b9ba0166-05bd-4f87-a976-05da618fa2e9" />

**Comment by potiuk** [comment]
> Resource section content is not displayed 

It's displayed. The main part of that issue was done. The remaining part is to make the "menu" "Community" to show up all the sections rather than them being show on the right side.

**Comment by gdivya6028** [comment]
@potiuk sure .. the content under Resources is not displayed on website

**Comment by potiuk** [comment]
Ah right. No idea why :)

**Comment by Satishmedarr** [comment]
Hi, I would like to work on this issue. Please assign it to me. I will add a “Resources” section to the Airflow website as requested. Thank you!

**Comment by srinjoymahanti** [comment]
Hi maintainers 👋
I’d like to work on this issue. Please assign it to me.

What I plan to do:

1. Locate the Community page source and identify the correct section placement (after “Join the community” and before “Primary Members Committers”)

2. Add a new “Resources” section with a clean layout (heading + short description + resource cards/links)

3. Include the mentioned resources like:

 Logos / promo materials from the Apache wiki link

 Airflow book link (Manning)

 Any other relevant community material if needed

4. Add the required menu/link entry on the Community page for easy navigation

5. Ensure styling matches the existing Airflow website theme and works well in desktop + mobile view

6. Open a PR with screenshots for review

I’ll start by reproducing the existing layout locally and then proceed with the changes. Thanks!

**Comment by potiuk** [comment]
Cool. Assigned you. One small thing. We should not link to the book, this is not a PMC-created and controlled thing, so we cannot really link to it from the "resources" page - this is what ASF does not really allow, the book is mentioned in the Ecosystem page - where we have clear disclaimer, those are links that are not PMC managed/endorsed/verified. So maybe instead liking to the book direcly, the resource page should mention ecosystem page and link to it - with similar disclaimer.

**Comment by srinjoymahanti** [comment]
@potiuk Thanks for assigning it! ✅
Got it — I won’t link directly to the book since it’s not PMC-managed/controlled.
Instead, I’ll update the new Resources section to point users to the Ecosystem page (where the disclaimer already exists) and I’ll add a similar disclaimer on the Resources page as well.

I’ll proceed with this approach and share the PR once ready. Thanks!

**Comment by srinjoymahanti** [comment]
Hi @potiuk  
As suggested, I avoided linking directly to the book since it’s not PMC-controlled.
Instead, the Resources section now points users to the Ecosystem page and includes a disclaimer that external links are not PMC managed/endorsed/verified.
PR: #1389 — please review when you get time. Thanks!

**Comment by potiuk** [comment]
Actually. I just see that we already have "Resources" section so there is not much to do.
